resources/analysisTools/indelCallingWorkflow/checkSampleSwap_TiN.pl

```diff
@@ -327,6 +341,11 @@
 
 `perl \${TOOL_NEW_COLS_TO_VCF} -vcfFile=$snvsGT_germlineRare_oVCF --newColFile=$snvsGT_germlineRare_oVCF.forAnnovar.temp --newColHeader=ANNOVAR_FUNCTION,GENE,EXONIC_CLASSIFICATION,ANNOVAR_TRANSCRIPTS --reportColumns=3,4,5,6 --bChrPosEnd=0,1,2 > $snvsGT_germlineRare_oVCF_annovar`;
 
+## Separating rare germline and rescued somatic varaiants
+`(cat $snvsGT_germlineRare_oVCF_annovar | grep '#' ; cat $snvsGT_germlineRare_oVCF_annovar | grep -P "Germline|SomaticControlRare" ) > $snvsGT_germlineRare_oVCF_annovar_rG`;
```

Comment:
Generally this seems to be fragile code:

Grep returns with exit code 1, if it cannot find a match. If the second grep does not yield any matches, then the whole expression may fail. This may or may not be tolerable and should get checked. Unfortunately, cat may also return exit code 1, if the input file is inaccessible. The expression can therefore not discriminate between tolerable exit code 1 and intolerable exit code 1. If there are infrastructure issues and files cannot be seen, this expression may just not not the problem.

In some situations such code may be fine, because the expressions are so simple they will never fail (except if the files are inaccessible due to infrastructure) and are used later in the same job where the error would be caught. Here, however, the files are not used anymore in the same job. If they are not used at all anymore, it may just be overlooked that they could not be written.

Therefore, add a test for the exit code (ideally also for the expressions in the old code).